### PR TITLE
Changes proposed in this pull request

### PR DESCRIPTION
**Pull request by estherchee**
- Update Navbar Styling
- Add and Style Footer

**Technical Requirement**
- Styling - Home Page

### CODE REVIEW

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+import HomePage from './pages/home/HomePage';
+import PostsPage from './pages/posts/PostsPage';
+import ChatPage from './pages/chat/ChatPage';
+import Footer from './components/Footer';
+import './styles/style.css';
 
 export default function App() {
   // Local State
-  const [loggedIn, setLoggedIn] = useState(Auth.isLoggedIn());
+  const [loggedIn, setLoggedIn] = useState(true);
```
Comment (janko101) [inline frontend/src/App.jsx]: Why do we use loggedIn state?

**Review by janko101 — COMMENTED** [review]
Hi Pei, it looks good. Can you just put some screenshots, so we can see how it looks, pls

File: frontend/src/App.jsx (hunk 1 above)
Comment (estherchee) [reply frontend/src/App.jsx]: It was supposed to be deleted. It was used to bypass the login process.

**Comment by estherchee** [comment]
Hi Team, I am closing pull request #23. 

Changes will be made on navbar, footer and background before we submit the next pull request. 

The loggedIn state will also be deleted.

**Comment by estherchee** [comment]
Hi Team, I am closing pull request #23.

Changes will be made on navbar, footer and background before we submit the next pull request.

The loggedIn state will also be deleted.

**Pull request closed without merging**